.github/workflows/gradle.yml

```
@@ -0,0 +1,31 @@
+
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ "users" ]
+  pull_request:
```

usersブランチがデフォルトブランチであればこの設定でもよいですが、もしいまPRを出しているこのブランチへのpushを契機にCIを実行したいだけなら、下記でよいです。

```suggestion
on:
  pull_request:
```

---

```
on:
  pull_request:
```
に修正致しました。

368678c6471745a5544c7741f286591dc97950b4

---

問題ないです！